spectrocloud/resource_cluster_edge_native.go

```diff
@@ -142,8 +143,15 @@ func resourceClusterEdgeNative() *schema.Resource {
 							Description: "List of public SSH (Secure Shell) to establish, administer, and communicate with remote clusters.",
 						},
 						"vip": {
-							Type:     schema.TypeString,
-							Optional: true,
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "The control plane endpoint can be specified as either an IP address or a fully qualified domain name (FQDN).",
```

Comment:
should say - vip can be specified instead of control plane endpoint. is FQDN supported?